packages/typespec-test/test/anomalyDetector/generated/typespec-ts/README.md

```diff
@@ -17,55 +19,34 @@ a kind of group based detection, this detection will find inconsistency ones in
-
-Key links:
-
-- [Package (NPM)](https://www.npmjs.com/package/@msinternal/ai-anomaly-detector)
+[Package (NPM)](https://www.npmjs.com/package/@msinternal/ai-anomaly-detector) |
 
 ## Getting started
 
 ### Currently supported environments
 
-- LTS versions of Node.js
+- [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule)
+- Latest versions of Safari, Chrome, Edge and Firefox.
 
-### Prerequisites
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
 
-- You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this package.
 
 ### Install the `@msinternal/ai-anomaly-detector` package
 
-Install the AnomalyDetector REST client REST client library for JavaScript with `npm`:
+Install the AnomalyDetector REST client library for JavaScript with `npm`:
 
 ```bash
 npm install @msinternal/ai-anomaly-detector
 ```
 
-### Create and authenticate a `AnomalyDetectorClient`
-
-To use an [Azure Active Directory (AAD) token credential](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-with-a-pre-fetched-access-token),
-provide an instance of the desired credential type obtained from the
-[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
 
-To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) 
 
-After setup, you can choose which type of [credential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) from `@azure/identity` to use.
-As an example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential)
-can be used to authenticate the client.
+### JavaScript Bundle
+To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our [bundling documentation](https://aka.ms/AzureSDKBundling).
 
-Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
-AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+## Key concepts
 
-## Troubleshooting
+### AnomalyDetectorClient
 
-### Logging
-
-Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
-
-```javascript
-const { setLogLevel } = require("@azure/logger");
-
-setLogLevel("info");
-```
+`AnomalyDetectorClient` is the primary interface for developers using the AnomalyDetector REST client library. Explore the methods on this client object to understand the different features of the AnomalyDetector service that you can access.
```

Comment:
why there's no other parts after `Key concepts` in this case?

Reply:
the package scope name is not azure or azure-rest